commentHandler.go

```diff
@@ -256,21 +263,78 @@ func manageLocking(req types.IssueCommentOuter, cmdType string) (string, error)
 	return buffer.String(), nil
 }
 
+func manageMilestone(req types.IssueCommentOuter, cmdType string, cmdValue string) (string, error) {
+
+	milestoneValue := cmdValue
+
+	var buffer bytes.Buffer
+
+	milestoneAction := strings.Replace(strings.ToLower(cmdType), "milestone", "", 1)
+
+	buffer.WriteString(fmt.Sprintf("%s wants to %s milestone of '%s' on issue #%d \n", req.Comment.User.Login, milestoneAction, milestoneValue, req.Issue.Number))
+
+	client, ctx := makeClient(req.Installation.ID)
+
+	allMilestones := &github.MilestoneListOptions{}
+	var milestoneNumber *int
+	var err error
+
+	theMilestones, _, milErr := client.Issues.ListMilestones(ctx, req.Repository.Owner.Login, req.Repository.Name, allMilestones)
+
+	if milErr != nil {
+		return buffer.String(), milErr
+	}
+
+	if cmdType == setMilestoneConstant {
+		if req.Issue.Milestone.Title != cmdValue && cmdType != "" {
+			for _, mil := range theMilestones {
+				if *mil.Title == milestoneValue {
+					milestoneNumber = mil.Number
+				}
+			}
+			input := &github.IssueRequest{
+				Milestone: milestoneNumber,
+			}
+
+			_, _, err = client.Issues.Edit(ctx, req.Repository.Owner.Login, req.Repository.Name, req.Issue.Number, input)
+		} else {
+			buffer.WriteString(fmt.Sprintf("Setting the milestone of #%d by %s was unnecessary.\n", req.Issue.Number, req.Comment.User.Login))
+			return buffer.String(), nil
+		}
+	} else {
+		_, err = nilMilestone(client, ctx, req.Issue.URL)
```

Comment:
you can rewrite this as:
```
if _, err := nilMilestone(client, ctx, req.Issue.URL); err != nil {
...
```

Reply:
What did you think of this suggestion @mrtind ?